packages/dw-react-controls/src/components/SplitPane/SplitPane.styles.ts

```diff
@@ -0,0 +1,71 @@
+import { ISplitPaneStyleProps, ISplitPaneStyles } from "./SplitPane.types";
+
+const GlobalClassNames = {
+	container: "dw-SplitPane-container",
+	leftPane: "dw-SplitPane-leftPane",
+	rightPane: "dw-SplitPane-rightPane",
+	separator: "dw-SplitPane-separator",
+	separatorLeft: "dw-SplitPane-separatorLeft",
+	separatorRight: "dw-SplitPane-separatorRight"
+};
+
+export const getStyles = (props: ISplitPaneStyleProps): ISplitPaneStyles => {
+	const { className, fixed, isDragging } = props;
+	const classNames = GlobalClassNames;
+
+	return {
+		container: [
+			className,
```

Comment:
shouldn't `className` be last, to ensure css specificity when overriding?

Reply:
True!